README.md

```diff
@@ -83,13 +83,13 @@ Participants will receive zoom invitations via email.
 
 | **Theme** | **Duration** | **UTC Time** |
 |-----------|--------------|--------------|
-| Second issue - break-out group discussion | 15 min | 20:00 - 20:15 |
-| Second issue - group reports | 10 min | 20:15 - 20:25 |
+| [Writing unit tests](activities/writing-unit-tests.md) | 15 min | 20:00 - 20:15 |
+| Second issue - break-out group discussion / group reports | 10 min | 20:15 - 20:25 |
 | Second issue - local development | 25 min | 20:25 - 20:50 |
 | Break | 10 min | 20:50 - 21:00 |
 | Second issue - submitting through issues/pull requests | 50 min |21:00 - 21:50 |
```

Comment:
Let's discuss here. I was thinking of moving the group discussion / group reports to be 20:15-20:50, and then do the local development after 21:00. May need to squeeze some of the activities after 21:00 to be shorter (did you have any material prepared for those?).

Reply:
That would be fine with me. I don't have any material prepared. My plan was for people to chose issues or other contributions to work on with us available to help (sort of like a mini-sprint). Does that sound good to you or would you prefer more structured material?

Reply:
Yep, a mini-sprint sounds good. How about we post on the Slack channel to find out what issues they would like to work on (i.e. with the eswn-workshop label, or any other they fancy), so we can assign issues to them.

Meanwhile, I'll try to squeeze some sections in the schedule together and you can comment on how it looks.

Reply:
Sounds great!